boards/riscv/nrf54h20pdk_nrf54h20/Kconfig.board

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
nit 3ca4a42f7b9aa99473114e3dd9d42b3b57a6a308 should be signed off by @57300  as well